### PR TITLE
Add printing output after wizard run

### PR DESCRIPTION
**Pull request by vz10**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/customizations/wizard/core.py [hunk 1]
```diff
@@ -222,9 +224,15 @@ def _evaluate_conditions(self, value, parameters):
...
+    def _fetch_parameters(self, value, parameters):
+        # in case parameters is a lazy evaluated dict-like object we need
+        # to fetch all the values to create key-value pairs for template
+        return {i[1]: parameters[i[1]]
+                for i in Formatter().parse(value) if i[1] is not None}
+
     def run_step(self, step_definition, parameters):
         value = self._evaluate_conditions(step_definition['value'], parameters)
-        return value.format(**parameters)
+        return value.format(**self._fetch_parameters(value, parameters))
```
Comment (kyleknap) [inline awscli/customizations/wizard/core.py:235]: Oh I've ran into this while working on another wizard. Instead, you can just update this to:
```python
value.format_map(parameters)
```
to get it working

File: awscli/customizations/wizard/wizards/iam/new-role-wip.yml [hunk 2]
```diff
@@ -214,7 +214,7 @@ execute:
         RoleName: "{role_name}"
         AssumeRolePolicyDocument: "{trust_policy}"
         Description: "{role_description}"
-      output_var: role_arn
```
Comment (kyleknap) [inline awscli/customizations/wizard/wizards/iam/new-role-wip.yml:217]: If we are going to rename the `role_arn` output variable, we will have to update it in the `write_config_file` step as well.

File: awscli/customizations/wizard/wizards/iam/new-role-wip.yml [hunk 3]
```diff
@@ -233,3 +233,17 @@ execute:
         # This role_arn comes from the `output_var` in the previous
         # executor step.
         role_arn: "{role_arn}"
+__OUTPUT__:
+  value: |
+    Wizard has created a role:
```
Comment (kyleknap) [inline awscli/customizations/wizard/wizards/iam/new-role-wip.yml:238]: For this first line, let's update it to `Wizard successfully created role with following properties:` to make it a little more explicit about what we are listing.

File: awscli/customizations/wizard/wizards/iam/new-role-wip.yml [hunk 4]
```diff
@@ -233,3 +233,17 @@ execute:
...
+      Role name: {role_name}
+      Role ARN: {created_role_arn}
+
+    {% if output_type == preview_cli_command %}
+    Example of AWS CLI commands:
```
Comment (kyleknap) [inline awscli/customizations/wizard/wizards/iam/new-role-wip.yml:243]: Then for these lines, we can then update it to be a little more explicit here as well:
* `Steps to create role is equivalent to running the following sample AWS CLI commands:`
* `Steps to create role is equivalent to deploying the following sample AWS CloudFormation template:`

**Review by kyleknap — COMMENTED** [review]
Looks good. Just had some comments.

File: awscli/customizations/wizard/core.py (hunk 1 above)
Comment (vz10) [reply awscli/customizations/wizard/core.py:235]: Yeah, totally forgot about this method.

**Review by kyleknap — APPROVED** [review]
Looks good to me. :ship:

**Pull request merged**